draft-ietf-tls-keylogfile.md

```diff
@@ -247,6 +247,12 @@ consumption by other programs.  In both cases, applications might require
 special authorization or they might rely on system-level access control to limit
 access to these capabilities.
 
+Forward secrecy guarantees provided in TLS 1.3 (see {{Section 1.2 and Appendix
+E.1 of ?RFC8446}}) and some modes of TLS 1.2 (such as those in {{Sections 2.2
+and 2.4 of ?RFC4492}}) do not hold if keys are saved.  Access to logged keys
+affects the authenticity, confidentiality, and integrity of any data that is
+exchanged for longer period than if an authentication key is compromised.
+
```

Comment:
```suggestion
Forward secrecy guarantees provided in TLS 1.3 (see {{Section 1.2 and Appendix
E.1 of ?RFC8446}}) and some modes of TLS 1.2 (such as those in {{Sections 2.2
and 2.4 of ?RFC4492}}) do not hold if key material is recorded.  Access to logged material allows an attacker to decrypt data exchanged in previous TLS connections and so undermine the intended security properties of ?RFC8446. 
```

Not sure what the second sentence was going for so word smithed a bit.

Reply:
```suggestion
Forward secrecy guarantees provided in TLS 1.3 (see {{Section 1.2 and Appendix
E.1 of ?RFC8446}}) and some modes of TLS 1.2 (such as those in {{Sections 2.2
and 2.4 of ?RFC4492}}) do not hold if key material is recorded.  Access to key
material allows an attacker to decrypt data exchanged in any logged TLS
connections.
```

?

Reply:
The aspect that I'm trying to bring out is that we're worried about an attacker getting a key log file in the present and then decrypting previously exchanged data they'd recorded (which is the thing that PFS stops). Your proposed second sentence gets rid of the nuance w.r.t. timing. Perhaps insert 'previously' before logged?

I have visions of these keylog files slowly filling up disks, not unlike nuclear waste, insofar as they remain hazardous indefinitely unless properly disposed of. I don't want to take up another round trip, so happy with whatever formulation you like best.